scripts/tag_latest_release.sh

```diff
@@ -177,14 +177,14 @@ done
 # Determine the result based on the comparison
 if [[ -z "$compare_result" ]]; then
     echo "Versions are equal"
-    echo "::set-output name=SKIP_TAG::true"
+    echo "SKIP_TAG=true" >> $GITHUB_OUTPUT
 elif [[ "$compare_result" == "greater" ]]; then
     echo "This release tag ${GITHUB_TAG_NAME} is newer than the latest."
-    echo "::set-output name=SKIP_TAG::false"
+    echo "SKIP_TAG=false" >> $GITHUB_OUTPUT
```

Comment:
@jongwooo Thanks for this fix! I think we may need to update this line [here](https://github.com/preset-io/superset/commit/4b77129cc9344a0e2eaabb54be1c99534ba2a0ad#diff-3cf86f687684a2c690521532de75a831def641338f5a9ec1da8add94714def4cR73) as well from `SKIP_TAG::false` to `SKIP_TAG=false`. Do you mind adding one more PR with this other change?

Reply:
@eschutho That'd be great. https://github.com/apache/superset/blob/ad6327db95ba8628e9890e2b2813ae088178d9c1/scripts/build_docker.py#L82 right?